lib/frameworks/jasmine2.js

```diff
@@ -86,6 +104,16 @@ exports.run = function(runner, specs) {
     return true;
   };
 
+  function isSuiteMatched(suite) {
+    if (!suite) {
```

Comment:
Yes. Its purpose is to skip suite checks and we have to return false then the spec filter will not consider the suite check.